src/ecs-deployment/index.ts

```diff
@@ -41,6 +41,13 @@ export interface EcsDeploymentProps {
    */
   readonly timeout?: cdk.Duration;
 
+  /**
```

Comment:
Please add a new integration test case for the scenario of including hooks in the deployment. See: https://github.com/cdklabs/cdk-ecs-codedeploy/blob/main/test/ecs-deployment.integ.ts

Reply:
Done. I think. This is the first time I've come across this type of integration tests. Is the idea just that it deploys cleanly, i.e. there are no assertions in the code against the deployed infrastructure?

Reply:
yep - thats correct. These tests create snapshots when tests are run against an AWS account and then the snapshots are verified on subsequent test runs.